ci_framework/roles/switch_config/tasks/onyx_config.yml

```diff
@@ -0,0 +1,113 @@
+---
+- name: Collect switch facts
+  mellanox.onyx.onyx_facts:
+  register: onyx_facts
+  failed_when: false
+
+- name: Check if JSON API is supported in switch
+  ansible.builtin.set_fact:
+    json_api_support: |-
+      {%- if onyx_facts['rc'] == 1 and 'command \"json-print\"' in onyx_facts['module_stderr'] -%}
+        {{ False | bool }}
+      {%- else -%}
+        {{ True | bool }}
+      {%- endif -%}
+
+- name: Print JSON API support
```

Comment:
Is this really needed? Printing a boolean that is evaluated in a the simplest assert task possible?

Reply:
I'll remove this debug